docs/pages/cdp/cdp-Installing.md

```diff
@@ -24,40 +24,28 @@ toc: false
     sudo npm install -g @brightside/core
     ```
 
-3. Build and install the cics-deploy plug-in as described in [Setting up your development environment](https://github.com/IBM/zowe-cli-cics-deploy-plugin/blob/master/docs-internal/tutorials/Setup.md). For example:
+3. Install the cics plugin:
 
     ```console
-    mkdir ~/cics-deploy
-    cd ~/cics-deploy
-    rm -Rf *
-    git clone https://github.com/IBM/zowe-cli-cics-deploy-plugin
-    cd zowe-cli-cics-deploy-plugin
-    npm install
-    npm run build
-    zowe plugins install .
+    zowe plugins install @brightside/cics
```

Comment:
Should we be using @brightside/cics or @zowe/cics ?

Reply:
`@brightside/cics@lts-incremental`

Reply:
done.
Changed it to @brightside/cics@lts-incremental